sklearn/preprocessing/data.py

```diff
@@ -2013,4 +2013,14 @@ def inverse_transform(self, X):
             mapping_func = interp1d(self.references_, quantiles_feat)
             Xt[:, feat_idx] = mapping_func(Xt[:, feat_idx])
 
-        return Xt
\ No newline at end of file
+        return Xt
+
+
+def quantile_normalize(X, axis=0, n_quantiles=1000, subsample=int(1e5),
+                       random_state=None):
+    n = QuantileNormalizer(n_quantiles=n_quantiles, subsample=subsample,
+                           random_state=random_state)
+    if axis == 0:
+        return n.fit_transform(X)
+    else:
+        return n.fit_transform(X.T).T
```

Comment:
This is something redundant in the module. Shall we open another PR with a common test for all the functions?